accounting/localizations/mexico.rst

```diff
@@ -257,74 +372,41 @@ If the invoice is not paid.
 **Legal considerations**
 
 - A cancelled invoice will automatically cancelled on the SAT.
-- If you retry to use the same invoice after cancelled, you will have as much
-  cancelled CFDI as you tried, then all those xml are important to maintain a
-  good control of the cancellation reasons.
-- You must unlink all related payment done to an invoice on odoo before
-  cancel such document, this payments must be cancelled to following the same
-  approach but setting the "Allow Cancel Entries" in the payment itself.
-
-
-Payments (Just available for CFDI 3.3)
---------------------------------------
-
-To generate the payment complement you only need to follow the normal payment
-process in Odoo, this considerations to understand the behavior are important.
-
-#. To generate payment complement the payment term in the invoice must be
-   PPD, because It is the expected behavior legally required for
-   "Cash payment".
-
-   **1.1. How can I generate an invoice with payment term `PUE`?**
+- If you retry to use the same invoice after cancelled, you will have as much cancelled CFDI as you tried, then all
+  those xml are important to maintain a good control of the cancellation reasons.
+- You must unlink all related payment done to an invoice on odoo before cancel such document, this payments must be
+  cancelled to following the same approach but setting the "Allow Cancel Entries" in the payment itself.
 
-   `According to the SAT documentation`_ a payment is classified as ``PUE`` if
-   the invoice was agreed to be fully payed before the 17th of the next
-   calendar month (the next month of the CFDI date), any other condition
-   will generate a ``PPD`` invoice.
+Credit Notes
+------------
 
-   **1.2. How can I get this with Odoo?**
+# todo
 
-   In order to set the appropriate CFDI payment term (PPD or PUE), you can
-   easily set it by using the ``Payment Terms`` defined in the invoice.
+Debit Notes
+------------
 
-   - If an invoice is generated without ``Payment Term`` the attribute
-     ``MetodoPago`` will be ``PUE``.
+# todo
 
-   - Today, if is the first day of the month and is generated an invoice with
-     ``Payment Term`` ``30 Net Days`` the ``Due Date`` calculated is going to
-     be the first day of the following month, this means its before the 17th
-     of the next month, then the attribute ``MetodoPago`` will be ``PUE``.
-
-   - Today, if an invoice is generated with ``Payment Term`` ``30 Net Days``
-     and the ``Due Date`` is higher than the day 17 of the next month the
-     ``MetodoPago`` will be ``PPD``.
+Payment in advance
+-------------------
 
-   - If having a ``Payment Term`` with 2 lines or more, for example
-     ``30% Advance End of Following Month``, this is an installments term,
-     then the attribute ``MetodoPago`` will be ``PPD``.
+# todo
```

Comment:
@nhomar we can remove this todo an add it on new PR once the documentation for Advance Payments is ready